manager/service.go

```diff
@@ -187,7 +187,7 @@ func (ms *managerService) Run(ctx context.Context, c *manager.ComputationRunReq)
 
 	ms.qemuCfg.VSockConfig.Vnc++
 
-	ms.publishEvent("vm-provision", c.Id, "complete", json.RawMessage{})
+	ms.publishEvent("vm-provision", c.Id, agent.Completed.String(), json.RawMessage{})
```

Comment:
also set a typed constant for manager states, i.e vm-provision, vm-running, ...

Reply:
We only have two states for manager  states, vm provision and stop computation

Reply:
there are additional states, see vm package and qemu package